design/ui/pagination/src/Pagination.module.scss

```diff
@@ -0,0 +1,87 @@
+._ {
+  display: grid;
+  grid-template-columns: 50px 1fr 50px;
+  grid-template-rows: 50px 1fr 50px;
+  gap: 0px 0px;
+  grid-template-areas: "hback header hfrwd" "content content content" "fback footer ffrwd";
```

Comment:
@lordpixel it breaks my heart... but until we are no longer supporting ie11... I think we might not be able to use CSS grid. 💔  It requires vendor prefixes to work there and does not fully support the whole spec. Are these CSS files the same as what that component is using in our production app?

PostCSS might be saving us there by adding those prefixes for us.